zmsentities/src/Zmsentities/AvailabilityHistory.php

```
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BO\Zmsentities;
+
+class AvailabilityHistory extends Schema\Entity
+{
+    public const PRIMARY = 'id';
```

PHPStan (2.2.7)

[error] Constant BO\Zmsentities\AvailabilityHistory::PRIMARY overriding constant BO\Zmsentities\Schema\Entity::PRIMARY (string) should also have native type string.


(classConstant.missingNativeType)